src/components/auction/AuctionBody/index.tsx

```diff
@@ -11,12 +11,14 @@ import { OrderBookContainer } from '../OrderbookContainer'
 import OrdersTable from '../OrdersTable'
 
 const Grid = styled.div`
-  display: grid;
+  display: flex;
+  flex-direction: column;
   grid-template-columns: 1fr;
```

Comment:
is it `grid-template-columns` valid in a flex container?

Reply:
extra css - fixed